light_gallery/migrations/0018_lightgallery_thumbnail_orientation.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-27 17:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('light_gallery', '0017_auto_20180822_1618'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='lightgallery',
+            name='thumbnail_orientation',
+            field=models.CharField(choices=[[b'v', b'Vertical'], [b'h', b'Horizontal (text on right)'], [b'hr', b'Horizontal (text on left)']], default=b'v', max_length=2, verbose_name='Orientation of thumbnail elements'),
+        ),
+    ]
```

Comment:
merge this migriation with the one above? Doesn't look like it needs to be broken out.